tests/unit/test_discretisations/test_discretisation.py

```diff
@@ -817,7 +811,7 @@ def test_broadcast(self):
             7 * np.ones_like(mesh["negative electrode"].edges[:, np.newaxis]),
         )
 
-    def test_broadcast_2D(self):
+    def test_broadcast_2_d(self):
```

Comment:
?
```suggestion
    def test_broadcast_2d(self):
```

Reply:
I think this was discussed in an earlier PR, it's a `pytestify` bug 🤔

Reply:
ahh, then please feel free to skip